WKElementaryDark.css

```diff
@@ -10,1873 +10,1873 @@
 ==/UserStyle== */
 
```

Comment:
The indents I was worried showed up after all. I don't mind them. My linux machine was also unhappy with the indentless state of the file.

Reply:
Can you provide examples of exactly what you're seeing? I'm unclear what the problem is. I use two-space indents and prettier with vscode. The file should render identically on osx or linux. I suspect that one of our platforms is inserting literal tab characters rather than spaces, and we need to NOT do that.

Reply:
Never mind. I see you meant that my one of our editors indented everything with spaces in the `@moz-document` block. I just checked and there don't appear to be any literal tab characters anywhere in the files.

Anyway, it looks better indented, so let's just bite the bullet once. It would be great if we both used the same formatter, then we could add, say, the `.prettierrc` file to source control to ensure we both format things with the same rules. I have vscode configured to run prettier on save.

Reply:
It's not an issue. I was just lamenting about inconsistency when it comes to stylus's editor across platforms. On windows, there are no indents, linux has them and I can only imagine so does macOS.

Use any formatter you would like and I will try to match it. I have been doing so since the univers and everything update anyhow 😄